rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @rentee = person
+  end
+
+  attr_accessor :date
```

Comment:
- You should add The accessor also for the book and person so you can modify and read them